.github/workflows/docker-and-k8s.yml

```diff
@@ -0,0 +1,110 @@
+name: Build agent packages.
+
+on:
+  workflow_call:
+    inputs:
+      image-type:
+        description: "Name of the image build type."
+        required: true
+        type: string
+      python-version:
+        required: false
+        type: string
+        default: 3.8.10
+      publish:
+        description: "If true, then the result image will be published on master branch or any tag."
+        type: boolean
+        required: false
+        default: true
+
+    secrets:
+      SCALYR_PROD_CLOUDTECH_TESTING_WRITE_TOKEN:
```

Comment:
May not be a bad idea to add description for those two secrets. 

At some point I may also rename them to reduce the confusing and use consistent naming format since both of them refer to the same Scalyr environment and account.